azure-pipelines.yml

```
@@ -78,3 +78,38 @@ jobs:
         ArtifactName: 'Windows Desktop Spanish'
       continueOnError: true
       condition: not(succeeded())
+
+- job: Ubuntu 16.04
```

Spaces aren't allowed in AZDO job names. This should get CI running.

```suggestion
- job: Ubuntu_16.04
```